docs/resources/custom_framework.md

```diff
@@ -0,0 +1,59 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "datadog_custom_framework Resource - terraform-provider-datadog"
+subcategory: ""
+description: |-
+  Manages custom framework in Datadog.
+---
+
+# datadog_custom_framework (Resource)
+
+Manages custom framework in Datadog.
+
+
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `handle` (String) The framework handle.
+- `name` (String) The framework name.
+- `version` (String) The framework version.
+
+### Optional
+
+- `description` (String) The description of the framework.
+- `icon_url` (String) The URL of the icon representing the framework.
+- `requirements` (Block Set) The requirements of the framework. (see [below for nested schema](#nestedblock--requirements))
+
+### Read-Only
+
+- `id` (String) The ID of the custom framework resource.
+
+<a id="nestedblock--requirements"></a>
+### Nested Schema for `requirements`
+
+Required:
+
+- `name` (String) The name of the requirement.
+
+Optional:
```

Comment:
It auto generates as optional even though I have a validator in the schema requiring controls